assets/app/scripts/filters/util.js

```diff
@@ -117,6 +117,9 @@ angular.module('openshiftConsole')
           }
           unit += (amount === "1" ? "core" : "cores");
           break;
+        case "storage":
+          unit = unit.substring(0, 1);
```

Comment:
Do we need `substring` at all? Any reason we can't reuse the memory case already defined above?